### PR TITLE
Add UpdateElectedHostosVersions and UpdateNodesHostosVersion proposals

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Extend NnsFunction enum with `UpdateElectedHostosVersions` and `UpdateNodesHostosVersion`.

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
We should wait until they are supported in nns dapp to merge, right?

**Comment by mstrasinskis** [comment]
> We should wait until they are supported in nns dapp to merge, right?

Just tested against the testnet.

**Pull request merged**
